bot/exts/help_channels/_cog.py

```diff
@@ -465,3 +488,30 @@ async def wait_for_dormant_channel(self) -> discord.TextChannel:
         self.queue_tasks.remove(task)
 
         return channel
+
+    async def update_available_help_channels(self) -> None:
+        """Updates the dynamic message within #how-to-get-help for available help channels."""
+        if not self.available_help_channels:
+            self.available_help_channels = set(
+                c for c in self.available_category.channels if not _channel.is_excluded_channel(c)
+            )
+
+        available_channels = AVAILABLE_HELP_CHANNELS.format(
+            available=', '.join(c.mention for c in self.available_help_channels) or None
+        )
+
+        if self.dynamic_message is not None:
+            try:
+                await self.bot.http.edit_message(
+                    constants.Channels.how_to_get_help, self.dynamic_message, content=available_channels
+                )
+                log.trace("Help channels have changed, dynamic message has been edited.")
```

Comment:
"Help channels have been updated" Sounds more accurate imo.

Reply:
Could you explain what you mean by "more accurate"? I don't really see the main purpose of changing between "changed" to "updated".

Reply:
I got the sentence wrong, nvm.